docs/db/week-3/lesson.md

```diff
@@ -58,9 +58,13 @@ const db = new Pool({
 });
 
 app.get("/customers", function (req, res) {
-  db.query("SELECT * FROM customers", function(err, result) => {
-    res.status(200).json({customers: result.rows})
-  })
+  db.query("SELECT * FROM customers")
+    .then((result) => {
+      res.status(200).json({ customers: result.rows });
```

Comment:
Refer to week3 line #713, shall we return an array or an object with `customers` properties holding the array?

Reply:
If it's an API response it should be an object of the form `{ resourceName : x }`. Is this what you mean?